plugins/node/opentelemetry-instrumentation-pino/test/pino.test.ts

```diff
@@ -243,7 +272,7 @@ describe('PinoInstrumentation', () => {
       instrumentation.enable();
     });
 
-    beforeEach(init);
+    beforeEach(() => init());
```

Comment:
this change is actually not needed, it just creates an unneded function

Reply:
There is a type mismatch. `beforeEach` accepts a function and can pass a callback as an argument to it. `init`'s optional argument is not a function.

Reply:
strange, at line 246 it seem to work. Anyhow, it's not important.